cypress/integration/menu/position.feature

```diff
@@ -1,42 +1,46 @@
 Feature: Position of a menu component
 
+    # default max width: 380px; default max height: 380px
     Background:
-        # the default maximum
-        Given the menu component has a height of 368px
-        # also the default maximum
-        And the menu component has a width of 380px
-        And the menu component is triggered by a user click on an anchor element
+        Given the menu component has a height and height of its default maximum
 
     Scenario: Default rendering
-        Given the anchor has more than 368px space below, before the end of the <body> element
-        When the menu is rendered
-        Then the menu is rendered below the anchor
+        Given there is enough space below the anchor to fit the default maximum
+        When the menu is opened
+        Then the menu is below the anchor
         And the menu is left aligned with the anchor
 
     Scenario: Flipped vertically
-        Given the anchor has less than 368px space below, before the end of the <body> element
-        When the menu is rendered
-        Then the menu is rendered above the anchor
+        Given there is not enough space below the anchor to fit the default maximum
+        When the menu is opened
+        Then the menu is above the anchor
         And the menu is left aligned with the anchor
 
-    Scenario: Less than 368px available space below and above anchor
-        Given the anchor has less than 368px space above and below before the limits of the <body> element
-        When the menu dropdown is rendered
-        Then it is rendered below the anchor
-        And the height of the menu is reduced to fit within the <body> element
-        But the heght of the menu is not reduced below 50px (and what if it is)?****
+    Scenario: Less than 368px and more than 50px available space below and above anchor
```

Comment:
I wonder how useful this would be. We're making sure that the element doesn't render off screen by making it scrollable. But, since there will have to be scrolling anyway to reveal the content, why not render it partially offscreen and let the user scroll the entire window to reveal the rest? Gets rid of an edge case for us.

If that might cause it to render off body, then it'd probably be best to render in the other direction. And otherwise just render off body. I'd prefer that over making a the component itself scrollable.

Reply:
That's what we were doing initially. @cooper-joe preferred to keep it inside the viewport.
I also think that a bit of scrolling won't be bothering the user, while a small scrollable box is not so nice on the other hand.

Reply:
Plus you have to scroll both, so there's not really a difference in the action required from the user.

Reply:
@cooper-joe ping

Reply:
An addition:
When reviewing on GitHub, the window to enter the review message and type of approval (comment, approved, request changes) is more or less in a menu as well and it just stays within the body. I much prefer that over it being re-positioned, I don't mind the scrolling at all:

![image](https://user-images.githubusercontent.com/10101831/69017873-7cd8d500-09e4-11ea-8d7b-dd24d511ca9f.png)

Reply:
Yes, I think we can leave out any kind of `viewport` measurements and just make sure these components fit within the `body`. As you said, it's easier to scroll the `body` than a small menu.